### PR TITLE
Release/7.0.0

### PR DESCRIPTION
**Pull request by PhungNg**
# Changelog

## Added
- Favicon section under Logotypes
- Copy to clipboard
- "For" attribute for Input fields
## Changed
- Tabs
  - Changed styling on component
  - Content on documentation page
- Change styling and documentation page on
  - Alert
  - Loader
  - Tabs
  - Links
- Changed content on
  - Checkbox
  - Radiobutton
  - Togglebox
  - Rangeslider
  - Select
  - Input field
- Changed options labels on Cards page
- Updated badges on pages

## Fixed
- Payex design guide
- Broken link in imagery

## Removed
- alert-complex class

### CODE REVIEW

File: src/less/documentation-payex.less [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+    p {
+        &.lead {
+            font-family: @brand-black;
+            font-weight: 500;
+        }
+    }
+}
+
+.documentation {
+    font-family: @brand-default;
```
Comment (VikingTristan) [inline src/less/documentation-payex.less:17]: This font-family rule should probably be moved outside of the `.documentation` class.

**Review by VikingTristan — COMMENTED** [review]
Whoah! Looks very nice.
A couple of nitpicks and comments here and there.
Perhaps we can fix the favicons before release since the favicons is the main "thing" about this release?

File: src/App/ComponentsDocumentation/components/Tabs/constants.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const TabsExample = ({ scroll, id }) => {
+    const items = scroll ? tabItems : tabItems.slice(0, 3);
+
+    return (
+        <div className="w-75">
+            <Tabs id={id} items={items} scroll={scroll}/>
+            <p>Content of the selected tabs</p>
+        </div>
+    );
+};
```
[the remark this answers is not in the document]
Comment (PhungNg) [reply src/App/ComponentsDocumentation/components/Tabs/constants.js:16]: Will be fixed on patch 7.0.1

File: src/App/Identity/identity/Logotype/index.js [hunk 3]
```diff
@@ -234,17 +213,30 @@ const HorizontalLogo = () => (
...
     <>
-        <h2 id="our-favicon">Our favicon</h2>
-        <ComponentPreview language="html" showCasePanel>
-            <img src={`${basename}icons/android-chrome-96x96.png`} alt="PayEx favicon"/>
-        </ComponentPreview>
+        <h2 id="our-favicon">Favicon</h2>
+        {brand === "payex"
+            ? <ComponentPreview language="html" showCasePanel>
+                <img src={`${basename}icons/android-chrome-96x96.png`} alt="PayEx favicon"/>
+            </ComponentPreview>
```
Comment (VikingTristan) [inline src/App/Identity/identity/Logotype/index.js:222]: Payex CLICK doesn't work

**Review by VikingTristan — APPROVED** [review]
Looking swell my mates

File: src/App/Identity/identity/Logotype/index.js (hunk 3 above)
Comment (VikingTristan) [reply src/App/Identity/identity/Logotype/index.js:222]: ok works now topkek

**Pull request merged**